DatadogSessionReplay/Sources/DatadogSessionReplay/Processor/SRDataModelsBuilder/WireframesBuilder.swift

```diff
@@ -82,11 +82,20 @@ internal class WireframesBuilder {
             )
         }
 
+        var fontSize = Int64(withNoOverflow: font?.pointSize ?? Fallback.fontSize)
+        if let boundingBox = textFrame?.size {
+            let area = boundingBox.width * boundingBox.height
+            let calculatedFontSize = Int64(sqrt(area / CGFloat(text.count)))
+            if calculatedFontSize < fontSize {
```

Comment:
What is the actual reasoning or observation behind this? It works great 🚀, but looks quite magic 🪄. How about adding code comment with some context / reference to this approach?